pkg/util/overridemanager/overridemanager_test.go

```diff
@@ -207,3 +208,261 @@ func TestGetMatchingOverridePolicies(t *testing.T) {
 		})
 	}
 }
+
+func Test_applyLabelsOverriders(t *testing.T) {
+	type args struct {
+		rawObj            *unstructured.Unstructured
+		commandOverriders []policyv1alpha1.CommandKeyValuePairOverrider
+	}
+	deployment := helper.NewDeployment(metav1.NamespaceDefault, "test")
+	deployment.Labels = map[string]string{
+		"for": "for",
+		"bar": "bar",
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    map[string]string
+		wantErr bool
+	}{
+		{
+			name: "test labels replace",
+			args: args{
+				rawObj: func() *unstructured.Unstructured {
+					deploymentObj, _ := utilhelper.ToUnstructured(deployment)
+					return deploymentObj
+				}(),
+				commandOverriders: []policyv1alpha1.CommandKeyValuePairOverrider{
+					{
+						Operator: policyv1alpha1.OverriderOpReplace,
+						Value: map[string]string{
+							"for": "test",
```

Comment:
Maybe we can add a new test for `non-exist` items. like `"nonexist": "nonexist"`.

Reply:
ok